annotate.py

```diff
@@ -180,7 +183,7 @@ def annotate_img(img_path, size1, size2, initials) :
 #       initials - First name and last name initials of the person doing the labelling. Recorded next to every PMT feature ID line in the text file.
 #       filename - The name of the .txt file that will be created.
 #Set up for directory of images with file structure for image segmentation
-def annotate_dir(img_dir, dataset, subset, size1, size2,initials,filename) :
+def annotate_dir(img_dir, size1, size2,initials,filename) :
```

Comment:
Since we're moving to a per-image output structure, `filename` can be removed here, and instead a text file for each image saved below.

For a future PR,  this function should in principle just be a loop calling `annotate_img` (or some new function that consolidates the copy/pasted code between the two functions).

Reply:
Yeap, that's issue #2, I will change the code to have `annotate_dir` call `annoate_img` first.

Reply:
right, forgot. make a new PR for that then